src/models/KeyboardModel.ts

```diff
@@ -156,6 +156,10 @@ class KeymapItem {
     this._curr.minus(x, y);
   }
 
+  hideKey() {
+    this.op.d = true;
+  }
```

Comment:
@adamrocker Should we think some side effect against we use the way to hide keys with the `d` flag? If we would want to show  a string with the `d` key in the future, the logic of this time will conflict for the `d` behavior.

Reply:
@yoichiro Agree with you. I'll delete the keys not hide them.

Reply:
@yoichiro I've fixed this in [this commit](https://github.com/remap-keys/remap/pull/643/commits/a89c29bae125bf4a03dd36c895efb45e0878c639). Could you please review it again?